lib/utils/find_files.rb

```diff
@@ -24,7 +35,7 @@ def find_files(path, opts = {})
     cmd += " -maxdepth #{depth.to_i}" if depth.to_i > 0
     cmd += " -type #{type}" unless type.nil?
 
-    result = inspec.run_command(cmd)
+    result = inspec.command(cmd)
     exit_status = result.exit_status
 
     return [nil, exit_status] unless exit_status == 0
```

Comment:
Currently we handle the exit status twice. Would could:
- just not return the `exit_status` if not required, only `files` or `[]`
- we return `[]` as default instead of `nil`

This would allow us to simplify the `find_files` method